Code/Tools/FBuild/Documentation/docs/functions/objectlist.html

```diff
@@ -54,15 +54,15 @@ <h1>ObjectList</h1>
   .CompilerInputUnity          ; (optional) Unity to build (or Unities)
   .CompilerInputAllowNoFiles   ; (optional) Don't fail if no inputs are found
   .CompilerInputObjectLists    ; (optional) ObjectList(s) whos output should be used as an input
-  
+
```

Comment:
It would be preferable to avoid unrelated whitespace changes if possible. There are quite a few in this review.

Reply:
👍I undid the whitespace changes.